sources/packages/backend/workflow/test/2021-2022/fulltime-assessment/eligibility/fulltime-assessment-2021-2022-eligibility-BCAG.e2e-spec.ts

```diff
@@ -98,7 +98,7 @@ describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}-eligibility-BCAG
         assessmentConsolidatedData.programCredentialType =
           CredentialType.GraduateDiploma;
         assessmentConsolidatedData.programLength =
-          ProgramLengthOptions.OneToTwoYears;
+          ProgramLengthOptions.SixtyWeeksToTwoYears;
```

Comment:
Minor. I would suggest to replace with `52WeeksTo59Weeks` as institution type must be the criteria here to determine BCAG to be not eligible

Reply:
Nice catch. Thanks!